frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -167,9 +193,7 @@ export const funnelLogic = kea<funnelLogicType>({
             },
         ],
         propertyCorrelations: [
-            {
-                events: [],
-            } as Record<'events', FunnelCorrelation[]>,
+            null as Record<'events', FunnelCorrelation[]> | null,
```

Comment:
q: Why do we need to make these nullable?

Reply:
To distinguish between actually having a response and not. Do you think that’s ok?

Reply:
For a loader for `results`, we have: `loadResultsSuccess` & `loadResultsFailure` & `resultsLoading` automatically generated. Hence, usually not needed?

Reply:
Don't mind it: it's just that it makes handling everywhere a bit more messier, since you have to start handling nulls

Reply:
Right, I guess having this as null forces anyone using this to consider that case, but I can change to using a separate loading prop?

Reply:
What's the use case here for checking it? From what I understand, we aren't doing things differently in either case?

Reply:
Yeah I'll remove. I can't recall the reason for the change, I think possibly so that I could put some guard in somewhere to avoid extra requests